.github/workflows/dockerimage.yml

```diff
@@ -7,11 +7,11 @@ on:
   release:
     types: [published]
   pull_request:
-    branches:
-      - master
+    paths:
+      - Dockerfile
+      - setup.py
```

Comment:
[Note] The trigger is not limited to the `master` branch, but I think it is acceptable because the built images are not uploaded to the DockerHub.